src/ontology/mondo-ingest.Makefile

```diff
@@ -128,6 +128,30 @@ $(COMPONENTSDIR)/icd10who.owl: $(TMPDIR)/icd10who_relevant_signature.txt | compo
 		remove -T config/properties.txt --select complement --select properties --trim true \
 		annotate --ontology-iri $(URIBASE)/mondo/sources/icd10who.owl --version-iri $(URIBASE)/mondo/sources/$(TODAY)/icd10who.owl -o $@; fi
 
+.PHONY: component-download-gard.owl
+component-download-gard.owl: | $(TMPDIR)
+	if [ $(MIR) = true ] && [ $(COMP) = true ]; then $(ROBOT) merge -I https://github.com/monarch-initiative/gard/releases/latest/download/gard.owl \
+	annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) -o $(TMPDIR)/$@.owl; fi
+
+# TODO: finish this goal
+# TODO: maybe back up some future version as commented out
+# TODO: @matentzn $(TMPDIR)/gard_relevant_signature.txt: Deactivated this prereq until gard-relevant-signature.sparql is created.
```

Comment:
`$(COMPONENTSDIR)/gard.owl`
@matentzn `$(TMPDIR)/gard_relevant_signature.txt`: Deactivated this prereq until `gard-relevant-signature.sparql `is created. Can you create this? Want me to make a GitHub issue for it and assign you?

Reply:
You do it, and select all of them, yes!